pub/css/main.css

```diff
@@ -60,27 +60,353 @@ textarea {
+}
+
+
+h3 {
+    max-width: 8em;
+    display: block;
+    font-size: 2.6em;
+    margin-right: 2em;
+    margin-left: 0.5ex;
+    color: #004f91;
+    font-weight: 400;
+    margin-bottom: 0;
+    font-family: 'Oswald', sans-serif;
+}
+
+.search a {
+    text-decoration: none;
+    color: black;
+    padding-right: 0.8ex;
+}
+
+.logo img {
+    height: 1em;
+    width: 1em;
+    border-radius: 50%;
+}
+
+.hidden {
+    display: none;
+}
+/* MAIN ARTICLE SECTION*/
+
+.date {
+    display: block;
+    color: #999;
+}
+
+.main {
+    width: 90%;
+    margin: 0 auto;
+    font-family: 'Open Sans', sans-serif;
+}
+
+#article-link {
+    color: #999;
+}
 
+blockquote {
+    border-left: solid #999 2px;
+    padding-left: 1em;
+    margin-left: 0;
+    font-size: 2em;
+    color: #004f91;
+    font-family: 'Oswald', sans-serif;
+}
+
+.share a {
+    height: 1.2em;
+    width: 1.2em;
+    text-decoration: none;
+    color: white;
+    background-color: #162876;
+    text-align: center;
+    line-height: 2.2ex;
+    margin: 1ex -0.1ex 1.5ex 1ex;
+    border-radius: 0.3ex;
+}
+
+.fa-share-square-o {
+    background-color: blue;
+    
+}
+
+/* SIDE COLUMN */
+
+aside {
+    text-align: center;
+    margin: 0 auto;
+    width: 23em;
+}
+.author-large {
+    height: 8em;
+    width: 8em;
+    border-radius: 50%;
+}
+
+section > h4 {
+    margin-bottom: -0.5ex;
+}
+
+.author-links {
+    text-decoration: none;
+    color: #999;
+    font-size: 90%;
+    padding: 1ex 1ex;
+}
+
+.about-ad {
+}
+.advertise {
+    height: 30ex;
+    width: 20em;
+    margin: 3ex auto 0 auto;
+    display: block;
+}
 
+.about-ad figcaption{
+    text-align: right;
+    padding-right: 1.5em;
+}
 
+.about-ad figcaption a {
+    color: #999;
+}
 
+/* POPULAR POST PICTURES */
 
+.pop-posts {
+    clear: both;
+    overflow: hidden;
+    background-color: #112740;
+    position: relative;
+    width; 100%;
+}
 
+.pop-pic {
+    position: relative;
+    display:block;
+    margin: 0 auto;
+    padding-right: 25%;
+    padding-bottom: 25%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    opacity: 0.5;
+    background-position: center center;
+    font-family: 'Oswald', sans-serif;
+    color: #112740;
+    text-decoration: none;
+    box-sizing: border-box;
+    
+}
 
+.pop-pic:hover {
+    opacity: 1;
+    color: white;
+    text-shadow: 2px 2px #000;
+}
 
+.pop-pic figcaption {
+    font-size: 2em;
+    position: absolute;
+    bottom: 0.2ex;
+}
+.pic1 {
+    background-image: url(http://stwww.surfermag.com/wp-content/uploads/2010/11/wpid-1kellyslater6772.jpg);
+}
 
+.pic2 {
+    background-image: url(http://food.mu-dev.lhblogs.com/files/flight-of-beers.jpg?26ca06)
+}
 
+.pic3 {
+    background-image: url(http://wac.450f.edgecastcdn.net/80450F/kyssfm.com/files/2014/01/Lounging-on-the-Beach-Credit-iStock-1671673501.jpg?w=600&h=0&zc=1&s=0&a=t&q=89);
+}
 
+.pic4 {
+    background-image: url(http://theplanetd.com/images/Croatia-Sea-kayaking-6-XL.jpg);
+}
```

Comment:
So these are more like id's than classes, since you're not likely to reuse them. I would probably drop `<img>` tags directly into the blocks rather than using a background image. Works pretty good your way, though...